iron-overlay-manager.html

```diff
@@ -22,6 +23,8 @@
 
     _backdrops: [],
 
+    _backdropElement: document.createElement('iron-overlay-backdrop'),
```

Comment:
Well that's fine, it made sense for me as well. But I think this is a bit of code smell. I mean there shouldn't be circular dependency between 2 elements, should there?